### DISCUSSION TITLE
关于多语言的逻辑

### DISCUSSION THREAD

**Discussion by guinanlin** [dream-num/univer#2442] [Q&A | 常见问题]
关于多语言的一点实现想法：

最近不是发布了多语言的处理方式的么， 我的实际使用场景是集成在vue3的项目里头。 
其实我建议， 你们可以发布语言包文件就可以。 
然后， 把界面的那些语言的设置的地方放在配置文件里头。 比如写在一个ts里头。 

对于我的这种场景的使用就变成这样子：
1， 需要自己维护自己的语言文件。 一般每个vue3项目都有自己的维护语言的集中的地方。
2， 维护之后， 我需要自己把内容对接到 ts 文件里头。 

这样子就实现了多语言的形式。 因为你们现在这种方式的话， 集成项目只能指定某种语言，
但是对于集成的项目而言， 语言需要做的是能切换，而且是跟随项目整体切换。

**Comment by jikkai** [comment]
Univer 的语言可以切换也可以从独立文件导出，请仔细阅读文档
